mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
No clue.  Straight up copy from what Ken did with takora.  He add the `os-release` source in https://github.com/ceph/ceph-build/commit/3bd7177ad9e6568c43a67eee023aa5d34ee5c6a0